config/initializers/99_clowder-config.rb

```diff
@@ -0,0 +1,43 @@
+
+if ClowderCommonRuby::Config.clowder_enabled?
+
+  config = ClowderCommonRuby::Config.load
+
+  # TO-DO: Refactor with a helper function (get_service_url(app_name, component)
+  host_inventory_hostname = config.private_dependency_endpoints.host-inventory.service.hostname
+  host_inventory_port = config.private_dependency_endpoints.host-inventory.service.port
+  host_inventory_url = "#{host_inventory_hostname}:#{host_inventory_port}"
```

Comment:
Nit: This might be a bit more readable:
```
host_inventory = config.private_dependency_endpoints.host-inventory.service
host_inventory_url = "#{host_inventory.hostname}:#{host_inventory.port}"
```

Reply:
Thank you, I appreciate the suggestion, and I've refactored those lines accordingly  :)